modules/tensor_mechanics/test/tests/ad_simple_linear/tests

```diff
@@ -14,7 +14,7 @@
     exodiff = 'linear-out.e'
     absent_out = '3 Nonlinear'
     prereq = 'linear-hand-coded'
-    requirement = 'We shall be able to reproduce the results of the hand-coded simulation using automatic differentiation'
+    requirement = 'We shall be able to reproduce the results of the hand-coded simulation using automatic differentiation in the production stress divergence kernel'
```

Comment:
What do you mean by 'in the production stress divergence kernel'? Do you mean `ADStressDivergenceTensors` (because that's what it looks like this test is using)?

Reply:
There used to be a testing Stress divergence kernel. This refers to the non-test kernel.